doc/troubleshooting.md

```diff
@@ -582,6 +582,32 @@ The name must not be blank.
 
 After correcting the configuration, the [changes will need to be propagated to the Yaskawa controller](../README.md#updating-the-configuration).
 
+### Alarm: 8011[23 - 54]
+
+*Example:*
+
+```text
+ALARM 8011
+ Failed creating/adding/init *something*
```

Comment:
Hmm ..

Could we maybe come up with something a bit nicer? :)

I agree we shouldn't duplicate this entry for all possible subcodes, but perhaps we can come up with a slightly nicer example text?

Reply:
```
ALARM 8011
 MotoROS2: Fatal Error
 Failed adding
 Failed to init
 Failed creating
```

?